docs/hopsworksai/aws/restrictive_permissions.md

```diff
@@ -23,20 +25,22 @@ After you have created the VPC either [Create a Security Group](https://docs.aws
 
 !!! note
     The [Security Group](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html#AddRemoveRules) and/or [Network ACLs](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-network-acls.html#Rules)
-    need to be configured so that at least port `80` is reachable from the internet otherwise creating Hopsworks instances will fail when creating SSL certificates.
+    need to be configured so that at least port `80` is reachable from the internet otherwise you will have to use self signed certificates in the hopsworks cluster.
```

Comment:
```suggestion
    need to be configured so that at least port `80` is reachable from the internet otherwise you will have to use self signed certificate in your Hopsworks cluster.
```